cmd/cmd.go

```diff
@@ -82,14 +82,12 @@ func StartService(cfg common.AresServerConfig, logger bark.Logger, queryLogger b
 		if controllerClientCfg == nil {
```

Comment:
the controller is not open source yet, right?

Reply:
not yet.
there's a way to enable/disable cluster mode, so the dependency on controller is optional.

Reply:
removed controller client, connect to zk directly instead